cmake/llvm-toolchain.cmake

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
This toolchain needs file-level documentation explaining its purpose and the semantics of its parameters (viz. BEMAN_BUILDSYS_SANITIZER)

Reply:
Done